index.js

```diff
@@ -1,28 +1,16 @@
 'use strict';
-const os = require('os');
-const EventEmitter = require('events');
 
-let NodeHookAddon;
-let currentPlatform = 'iohook_';
-
-// TODO need make prebuilds for Electron users, but it can require too many prebuilds
-if (process.versions['electron']) {
-  currentPlatform += 'electron.' + process.versions['electron'] + '_';
-}
-
-currentPlatform += os.platform() + '_' + os.arch();
-// console.log('Used platform', currentPlatform);
-try {
-  NodeHookAddon = require("bindings")(currentPlatform);
-} catch (e) {
-  console.log('Try use default');
-  NodeHookAddon = require("bindings")('iohook');
-}
+var util = require('util');
+var EventEmitter = require('events');
+var runtime = process.versions['electron'] ? 'electron' : 'node'
+var essential = runtime + '-v' + process.versions.modules + '-' + process.platform + '-' + process.arch
```

Comment:
Not sure if "process.versions.modules" (which gives the ABI) and process.platform (also available as os.platform()) and process.arch (maybe also available as os.arch() ?) will be available on all different platforms, seemed to be available on node.js and NW.js though.

Perhaps this should be overridable by a config / environment variable?